scripts/linters/pre_commit_linter.py

```diff
@@ -452,6 +476,7 @@ def main(args=None):
     # Default mode is non-verbose mode, if arguments contains --verbose flag it
     # will be made True, which will represent verbose mode.
     verbose_mode_enabled = bool(parsed_args.verbose)
+    linter_type_used = _get_linter_type(parsed_args.linter_type)
```

Comment:
I would suggest having this maybe:
```
# Setting default value for linters to use.
linter_types_to_use = [CUSTOM_LINTER_TYPE, THIRD_PARTY_LINTER_TYPE]
if parsed_args.linter_type is not None:
    linter_types_to_use = [parsed_args.linter_type]
```
[I'm not very confident with the name of arg `linter_type` and `linter_types_to_use`. @seanlip, do you have any suggestion?

[Also, add the default behavior for linter types in the top docstring and parser help section.]

Reply:
another approach:
```
_PARSER.add_argument(
     '--linter-types',
    nargs='+',
    choices=['third_party', 'custom'],
    help='type of linter to be used',
    action='store_custom',
    custom=[CUSTOM_LINTER_TYPE, THIRD_PARTY_LINTER_TYPE])
```
This will set the default value to both the linters and will accept multi linter types through command line*

I'm fine with either [but with a better name maybe?]

Reply:
Yeah, I agree, the naming is a bit confusing. @Hudda I suggest you just inline the logic here.



Also, in general, start varnames with the type, e.g. LINTER_TYPE_CUSTOM, LINTER_TYPE_THIRD_PARTY. That makes it a bit easier to see which linter types fall in the same "set".

Reply:
Done.